test/gcollections.cpp

```diff
@@ -79,7 +79,7 @@ struct Heap<C, true> {
 };
 
 template <typename C>
-void testBasic(std::string prefix, C&& collection, size_t N) {
+void testBasic(std::string prefix, C&& collection, int N) {
```

Comment:
Anything that ever represents a "collection size" should always be size_t, but I suspect this patch is intended to  make the test match what the data structure does to remove a warning, so we can switch everything over to size_t later.